addons/packages/app-toolkit/0.1.0/README.md

```diff
@@ -0,0 +1,141 @@
+# (Experimental) Application Toolkit
+
+Application Toolkit package is a meta-package that helps user install a collection of packages for creating, iterating and managing applications
+
+## Supported Providers
+
+The following table shows the providers this package can work with.
+
+| Docker |
```

Comment:
Does this package only work on Docker? I've only tested locally with an unmanaged-cluster, should we try AWS or Azure?